bedrock/foundation/templates/foundation/leadership-network.html

```diff
@@ -0,0 +1,63 @@
+
+{% block page_title %}{{ _('The Mozilla Leadership Network') }}{% endblock %}
+
+{% set body_id = "leadership-network" %}
+
+{% block article_header %}
+  <h2>{{ _('The Mozilla Leadership Network') }}</h2>
+  <p>{{ _('Connecting Leaders') }}</p>
+{% endblock %}
+
+{% block article %}
+
+  <p>
+    {% trans %}
+    The Mozilla Leadership Network is a global network of diverse leaders - from the young activist teaching web literacy to her community, to the executive leading a global organization - who will ensure the next wave of access, inclusion and opportunity online.
+    {% endtrans %}
+  </p>
+
+  <p>{{_('Leaders in this network will take action together to:') }}</p>
+  <ul>
+    <li>{{ _('Shape and spread the open internet agenda;')}}</li>
+    <li>{{ _('Model ‘open’ values and practices in institutions, products and policies around the world.')}}</li>
+  </ul>
+
+  <h3>{{_('How we work')}}</h3>
+  <p>
+    {% trans %}
+    The Mozilla Leadership Network accomplishes its work through thematic ‘hubs,’ communities of people with diverse expertise, who are helping others build the capacity to advance the open Internet in their field or topic of interest.
+    {% endtrans %}
+  </p>
+
+  <p>{{_('Our current hubs include:')}}</p>
+
+  <ul>
+    <li>{% trans url_learning="https://learning.mozilla.org" %}<a href="{{ url_learning }}">Learning.</a> Educators and librarians committed to transforming learning in our digital age.{% endtrans %}</li>
+    <li>{% trans url_science="https://science.mozilla.org" %}<a href="{{ url_science }}">Science.</a> Researchers and data scientists committed to making research and practice more open.{% endtrans %}</li>
+    <li>{% trans url_advocacy="https://advocacy.mozilla.org" %}<a href="{{ url_advocacy }}">Internet Policy & Advocacy.</a> Individuals and organizations focused on advancing open Internet issues.{% endtrans %}</li>
+    <li>{% trans url_women="https://mozilla.github.io/womenandweb/" %}<a href="{{ url_women }}">Women & Web Literacy.</a> Community organizers and senior leaders creating opportunities for women and girls to develop web literacy skills.{% endtrans %}</li>
+    <li>{% trans url_iot="https://github.com/openiotstudio" %}<a href="{{ url_iot }}">Internet of Things.</a> Hackers and designers creating provocative prototypes for what an open Internet of Things could and should be.{% endtrans %}</li>
+  </ul>
+
+  <h3>{{ _('Our Initiatives') }}</h3>
+  <p>
+    {{ _('The Mozilla Leadership Network pursues our goals through five key initiatives, which are reflected in each hub:')}}
+    <ul>
+      <li>{% trans url_weblit_module="https://thimbleprojects.org/chadsansing/75057/#overview" %}<em>Curricula &</em> Training. We provide free and open curricula and training programs to help leaders develop the skills they need to ensure the next wave of access, inclusion and opportunity online. Example: <a href="{{ url_weblit_module }}">Web Literacy Training Module</a>{% endtrans %}</li>
```

Comment:
Sorry I missed this before, but I imagine the `</em>` should go after "Training." here